doc/docs/master/concepts/data-concepts/index.md

```diff
@@ -25,22 +25,26 @@ run into a merge conflict with Pachyderm.
 The Pachyderm data versioning system has the following main concepts:
 
 **Repository**
-:   A Pachyderm repository is the highest level data object. Typically,
-    each dataset in Pachyderm is its own repository.
+:   A Pachyderm repository is the highest level data object,
+    an independent file system. Typically, each dataset in
+    Pachyderm is its own repository.
 
 **Commit**
-:   A commit is an immutable snapshot of a repo at a particular point
-    in time.
+:   A commit is an immutable snapshot of a data in Pachyderm corresponding
```

Comment:
feels superfluous to have "sub-commit" as a top level entry here, but maybe that helps call it out more clearly?

Reply:
Added the following issue to come back to that definition: #6760